src/main/java/gov/usgs/locator/LocMain.java

```diff
@@ -149,15 +167,25 @@ public static void main(String[] args) {
       } else if (arg.startsWith(CSVFILE_ARGUMENT)) {
         // get csv file
         csvFile = arg.replace(CSVFILE_ARGUMENT, "");
+      } else if (arg.startsWith(LOCCONFIG_ARGUMENT)) {
+        // get locator configuration
+        locationConfigPath = arg.replace(LOCCONFIG_ARGUMENT, "");
       }
     }
 
-    if ("json".equals(fileType)) {
+    if ("json".equals(inputType)) {
       inputExtension = ".locrequest";
-      outputExtension = ".locresult";
+    }
+    if ("detection".equals(inputType)) {
+      inputExtension = ".jsondetect";
     } else {
       inputExtension = ".txt";
+    }
+
+    if ("hydra".equals(outputType)) {
```

Comment:
does this mean that json formatted outputs have a file extension of .locresult? maybe the file names is a legacy thing, but usually I expect a file extension to help identify the format of the data in the file, so I would *think* hydra outputType would have an extension of .hydra and json outputType would have a file extension of .json, but it seems more is happening here, so maybe worth documenting as a comment? or...?